### PR TITLE
Added getEntrypointPath to ChunkExtractor and reading json from fs

### PR DESCRIPTION
**Pull request by fivethreeo**
#950

### CODE REVIEW

**Review by theKashey — APPROVED** [review]
Very good move 👍

File: packages/server/src/ChunkExtractor.js [hunk 1]
```diff
@@ -205,7 +205,7 @@ class ChunkExtractor {
     inputFileSystem = fs,
```
Comment (theKashey) [inline packages/server/src/ChunkExtractor.js:205]: 🤔 🤔 🤔 this symbol is __undefined__

Comment (fivethreeo) [reply packages/server/src/ChunkExtractor.js:205]: Undefined? fs is imported. Errors?

Comment (theKashey) [reply packages/server/src/ChunkExtractor.js:205]: 🤦‍♂️ imported above the fold of github review

**Comment by theKashey** [comment]
👍 👍 👍

**Pull request merged**